src/api/index.ts

```diff
@@ -1,11 +1,14 @@
-export const fetchAPI = async (url: string) => {
+import { BASE_URL } from "@constants";
+
+export const fetchAPI = async <T>(url: string): Promise<T> => {
   try {
-    const response = await fetch(url);
+    const response = await fetch(`${BASE_URL}${url}`);
```

Comment:
```suggestion
    const response = await fetch(`${BASE_URL}/${removeFirstSlash(url)}`);
```
요렇게 해주면 `url` 변수에 슬래시가 있든 없든 항상 일관된 동작을 보장해줄 수도 있겠네요